volatility3/framework/plugins/windows/direct_system_calls.py

```diff
@@ -0,0 +1,455 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
These seem new this year?  They should say their creation date for the copyright, so 2024 please.

Reply:
Fixed